src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  flex-shrink: 0;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
```

Comment:
`#fff ` is duplicated, so make sense to move to the variable
please move all duplicated styles to the variables